src-webviews/src/pages/job/Job.vue

```diff
@@ -8,6 +8,18 @@
                 <img :src="ResolvePath(image)" style="width: 100%" />
             </div>
             <p class="body-2" v-html="getSummary"></p>
+
+            <div class="split split-full center pt-4" v-if="maxAmount !== undefined">
```

Comment:
This actually shouldn't be in job trigger at all. This probably needs to be its own new Vue file. To prevent conflicts with other job triggers.